src/zap/zapinfo.cpp

```diff
@@ -4497,6 +4497,12 @@ bool ZapInfo::canTailCall(CORINFO_METHOD_HANDLE caller,
                                          CORINFO_METHOD_HANDLE exactCallee,
                                          bool fIsTailPrefix)
 {
+#ifdef FEATURE_READYTORUN_COMPILER
+    // READYTORUN: FUTURE: Delay load fixups for tailcalls
+    if (IsReadyToRunCompilation())
+        return false;
```

Comment:
I thought we are enabling tail calls for CoreRT.  But Won't this disable it?

I guess, it is not clear to me whether in the context of CoreRT  IsReadyToRunCompilation() will be true.